test/dynamo/test_dynamo_aliasing.py

```diff
@@ -153,7 +153,7 @@ def dummy_inplace(input):
 
     device = xm.xla_device()
     input = torch.randn(5, 5).to(device)
-    input_cloned = torch.clone(input)
```

Comment:
So, `clone` cannot be used in pt-xla?

Reply:
clone can be used, but in this test we manually donated `input`'s buffer, which is shared by `input_cloned`. 

actually I made this change prior to my fix, I think cloned should work even with manual buffer donation.. through it is generally a bad idea to donate the buffer while also clone the same tensor..